src/main.js

```diff
@@ -1,19 +1,12 @@
-const { terminate } = require('./app/utils');
-const db = require('./db');
-const app = require('./app');
-const port = process.env.PORT || 3000;
+const startApp = require('./app');
+const config = require('./config');
+const environment = config.NODE_ENV || 'development';
+const seeds = require('./sequelize.config')[environment].custom.seeds;
 
 (async () => {
-  await db.connect();
-  await db.reset();
-  const server = app.listen(port, () => {
-    console.log(`App is listening on port ${port}`);
-  });
-
-  const exitHandler = terminate(server);
-
-  process.on('uncaughtException', exitHandler(1, 'Unexpected Error'));
-  process.on('unhandledRejection', exitHandler(1, 'Unhandled Promise'));
-  process.on('SIGTERM', exitHandler(0, 'SIGTERM'));
-  process.on('SIGINT', exitHandler(0, 'SIGINT'));
+  for (const seedsPath of seeds) {
+    await require(seedsPath)();
+  }
```

Comment:
this must be executed in scope of additional script.
It is possible that in prod you won't need to run seeding.

Reply:
you can create `scripts` folder and store there all scripts which are not directly related to app's workflow.

Reply:
see https://12factor.net/